subseas_flex_fcst/maproom.py

```diff
@@ -267,11 +306,15 @@ def local_plots(marker_pos, start_date, lead_time):
         )
     )
     cdf_graph.update_traces(mode="lines", connectgaps=False)
+    if CONFIG["time_units"] == "days":
+        start_date_pretty = (pd.to_datetime(start_date)).strftime("%-d %b %Y")
+    else:
```

Comment:
I would prefer to check for "months" (or whatever) explicitly, and raise an exception if the value of `time_units` is none of the expected values.

Reply:
I did something that doesn't raise an exception. I didn't want to break everything just because we can't get the right text printed. What do you think?